zgw/objectenapi/src/main/kotlin/nl/nlportal/zgw/objectenapi/domain/CreateObjectsApiObjectRequest.kt

```diff
@@ -29,4 +29,16 @@ data class CreateObjectsApiObjectRequestRecord<T>(
     val startAt: String,
     var correctionFor: String? = null,
     var correctedBy: String? = null,
+)
+
+data class CreateObjectsApiObjectRequestWithoutCorrection<T>(
```

Comment:
Waarom een nieuwe class, want volgens mij kan je ook gewoon CreateObjectsApiObjectRequest gebruiken, want correction properties mogen null zijn

Reply:
De obejcts api accepteert het object niet als er een call word gedaan met een null in de correction for/by